scripts/setupVitest.ts

```diff
@@ -1,4 +1,4 @@
-import { vi } from 'vitest'
+/// <reference types="vitest/globals" />
```

Comment:
Could you please resolve conflicts with this PR? 

Also `/// <reference types="vitest/globals" />` is no longer needed, since we already defined it in `tsconfig.json`

https://github.com/vuejs/core/blob/1f83020a2baad1588c964b6dabc819a22bb43b94/tsconfig.json#L22